edx-platform/pearson-vue-theme/lms/templates/courseware/tabs.html

```diff
@@ -13,14 +14,17 @@
     return "active"
   return ""
 %>
+
+<%ALTERNAVITE_STRINGS = configuration_helpers.get_value('ALTERNAVITE_STRINGS', {})%>
```

Comment:
Is this necessary here? Are you going to use ALTERNATIVE_STRINGS? @luismorenolopera